apparmor.d/profiles-m-r/rustdesk

```diff
@@ -0,0 +1,143 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{,usr/}{,local/}bin/rustdesk
```

Comment:
Why `local`? Because it's a bleeding edge software which is absent in the package repository.